src/Bridge/Doctrine/Orm/Extension/FilterEagerLoadingExtension.php

```diff
@@ -115,12 +116,14 @@ private function getQueryBuilderWithNewAliases(QueryBuilder $queryBuilder, Query
 
         //Change join aliases
         foreach ($joinParts[$originAlias] as $joinPart) {
+            /** @var Join $joinPart */
+            $joinString = str_replace($aliases, $replacements, $joinPart->getJoin());
+            $alias = substr($joinString, 0, strpos($joinString, '.'));
+            $association = substr($joinString, strpos($joinString, '.')+1);
```

Comment:
can we avoid multiple calls on strpos?

Do you think it's possible that strpos returns `false` here? Maybe we kinda need a test?

Reply:
@soyuka As it's from a cloned query builder, I don't think it's even possible that `strpos` could return `false`. It would mean that the original query builder is invalid, which report the bug to doctrine.